### ISSUE TITLE
finish unit tests

### ISSUE THREAD

**Issue by florianhartig** [biometry/RLPJ#42]
Hi Ramiro,

I have included the test folder structure in the unit tests and started a few simple tests - could you maybe see if you could add more? I found it a bit difficult to write the tests because the help is sometimes not 100% clear how stuff works. I'll write an extra ticket about this

**Comment by Rasilgon** [comment]
Hi Florian, 
I only see the file test-runLPJ-outputTypes.R and one test for plotLPJData. Are there any other tests that you have not pushed yet?

I could write basic tests (function works), as the one for plotLPJData,  for the functions:

- writeTemplate
- getLPJData
- setupLPJParallel

and also for the utility functions: getTemplate, getParameterList, getDesign, etc.

**Comment by florianhartig** [comment]
no, I haven't pushed any other test yet. If you have time to do this that would be great, I'm swamped at the moment.
